onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -1593,6 +1594,84 @@ activation.)DOC")
                                   ONNX_NAMESPACE::convPoolShapeInference(ctx, true, false, 0, 1);
                                 }));
 
+ONNX_MS_OPERATOR_SET_SCHEMA(FusedConvTranspose, 1,
```

Comment:
Please add C++ unit tests so the schema/shape inferencing/etc. is validated in a basic build (i.e. CPU EP only).

Reply:
Is there an example test for this ?

Reply:
Not applicable anymore as FusedConvTranspose change is reverted.